src/main/java/com/eta/houzezbackend/controller/AgentController.java

```diff
@@ -40,6 +40,12 @@ public Agent activeAgent(@RequestParam String token, @PathVariable Long id) {
         return agentService.setAgentToActive(token);
     }
 
+    @PatchMapping("/reset-password")
+    public AgentGetDto resetPassword(@Valid @RequestBody AgentSignUpDto agentSignUpDto) {
```

Comment:
Don’t you think the password can be reset by anyone?

Reply:
Got it. Frontend pass me the token and new password.